runtime/ops/net.rs

```diff
@@ -64,9 +61,7 @@ pub struct OpConn {
 pub enum OpAddr {
   Tcp(IpAddr),
   Udp(IpAddr),
-  #[cfg(unix)]
   Unix(net_unix::UnixAddr),
-  #[cfg(unix)]
   UnixPacket(net_unix::UnixAddr),
```

Comment:
Unixpacket is not supported on Windows.

Reply:
Thanks, I suspected that might not be the case. I don't think I've seen `unixpacket` used anywhere TBH (not just deno, but any software project)

Reply:
Probably going to get a lot more common with QUIC / HTTP/3.